tools/validators/ontology_validator/yamlformat/tests/presubmit_validate_types_lib_test.py

```diff
@@ -228,6 +228,28 @@ def testConfigUniverseGetUnitsForMeasurement(self):
         'discharge_fan_lost_power_alarm')
     self.assertIsNone(units)
 
+  def testConfigUniverseGetUnitsForMeasurementMultipleNoUnits(self):
+    doc = {
+      'powerfactor': [{'no_units': 'STANDARD'}],
```

Comment:
just to make this test more complete can you please use two units one standard and non-standard for powerfactor: no_units, another_one and update the test accordingly